spec.md

```diff
@@ -565,15 +565,18 @@ message VolumeMetadata {
 }
 
 // A standard way to encode credential data. The total bytes of the values in
-// the Data field must be less than MaxSecretSize bytes.
+// the Data field must be less than 1 Mebibyte.
```

Comment:
This looks like a very randomly picked limit :) I don't have a strong feeling on this. I think having restriction is better than not having it. Removing a restriction is always easier than adding a restriction later.

Just for the sake of consistency, should we pose a restriction on all the fields in this spec?